doc/topics/releases/oxygen.rst

```diff
@@ -281,6 +281,54 @@ Salt Minion now no longer need to be configured against a specifig DNS name or I
 For this feature Salt Master now requires port 4520 for UDP broadcast packets to be opened
 and the Salt Minion be able to send UDP packets to the same port.
 
+Configuration
+=============
+
+By default, automatic discovery is disabled.
+
+Warning: due to the current limitations that will be changing in a future, before you turn on autodiscovery,
+make sure your network is secured and trusted. See the limitations below.
+
+Autodiscovery is configured on Master and Minion. Both of them are configured via ``discovery`` option
+as follows:
+
+**Master configuration**
+
+To simply enable it by default, accepting all kind of minions, is enough to set it to True:
+
+.. code-block:: yaml
+
+       discovery: true
+
+Sub-option `mapping` allows autodiscovery help Minions to find a proper Master for them. The mapping contains
+just an arbitrary set of key/value strings that would match either any of them or all of them. Default is empty.
+
+Example:
+
+.. code-block:: yaml
+
+       discovery:
+         mapping:
+           description: SES 5.0
+           node: 1
+
+More of this below.
+
+It is also possible to change default ``4520`` port, by setting another integer value to the ``port`` option within
+the discovery system. However, this should be also pre-configured on the minions as well.
```

Comment:
```
It is also possible to change the port used from the default of ``4520``, by
setting a ``port`` option under the Master's ``discovery`` configuration:

.. code-block:: yaml

    discovery:
      port: 4567

.. note::
    When using a port number other than the default, the Minion's ``discovery``
    configuraton must *also* have a port specified, otherwise the Minion will
    still attempt to contact the Master on port ``4520``.
```